src/EventProducerService.cpp

```diff
@@ -62,16 +56,16 @@ void EventProducerService::process(UserInterface::RequestedAction requestedActio
   }
 }
 
-void EventProducerService::sendEvent(bool state, byte evValue)
+void EventProducerService::sendEvent(bool state, byte index)
 {
   byte nn_en[4];
-  byte index;
   byte opCode;
-  index = module_config->findExistingEventByEv(1, evValue);
+
   if (index < module_config->EE_MAX_EVENTS)
   {
     module_config->readEvent(index, nn_en);
-    unsigned int nn = ((nn_en[0] << 8) && nn_en[1]);
+    unsigned int nn = ((nn_en[0] << 8) + nn_en[1]);
```

Comment:
Please note that I changed this recently to use an if statement like "if ((nn_en[0] == 0) && (nn_en[1] == 0))"
This actually saves a few bytes.
nn is only used in the if-statement below (and the debug but that one can use the long expression).
You may get a merge conflict on this change.

Reply:
That's simpler.  I'll change it.